spotlight/losses.py

```diff
@@ -242,7 +242,7 @@ def poisson_loss(observed_ratings, predicted_ratings, weights=None):
 
     assert_no_grad(observed_ratings)
 
-    loss = (predicted_ratings - observed_ratings) * torch.log(predicted_ratings)
+    loss = predicted_ratings - observed_ratings * torch.log(predicted_ratings)
```

Comment:
Was your change intentional? I'm not entirely sure if what it was was right, but this change certainly made the loss worse in the unit test.